test/e2e/util.go

```diff
@@ -38,6 +38,7 @@ func NewUPSBrokerPod(name string) *corev1.Pod {
 					Args: []string{
 						"--port",
 						"8080",
+						"-alsologtostderr",
```

Comment:
ah I see that you do add a `-alsologtostderr` flag too there. We probably need to add it to the Helm chart ( https://github.com/kubernetes-incubator/service-catalog/blob/master/charts/ups-broker/templates/broker-deployment.yaml)?
Not related to this PR though.